client/extensions/hello-dolly/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { navigation, siteSelection } from 'my-sites/controller';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import Main from 'components/main';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+
+const render = ( context ) => {
+	renderWithReduxStore( (
+		<Main className="hello-dolly__main">
+			<SectionHeader label="Hello, Dolly!">🐑</SectionHeader>
+			<Card>
+				<p style={ { fontSize: 18, fontWeight: 300 } }>This is not just an extension, it symbolizes the hope and enthusiasm of an entire generation summed up in two words sung most famously by Louis Armstrong.</p>
```

Comment:
Nit: could we split these to multiple lines so it can be read without having to scroll horizontally?